### PR TITLE
LINK-2104 | Prevent to delete signup after event start_time

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Prevent to delete a signup or a signup group after an event has started.

## Closes
[LINK-2104](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2104)

[LINK-2104]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2104?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr168-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr168-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr168-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr168-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: public/locales/en/signup.json [hunk 1]
```diff
@@ -124,6 +124,10 @@
     "capacityInWaitingList_other": "Registration for this event is still possible, but there are only {{count}} seats left in the queue.",
     "capacityInWaitingListNoLimit": "Registration for this event is still possible, but there are only seats left in the queue.",
     "closed": "Registration for this event is currently closed. Please try again later.",
-    "closedWithEnrolmentTime": "Registration for this event opens on {{openingDate}} at {{openingTime}}."
+    "closedWithEnrolmentTime": "Registration for this event opens on {{openingDate}} at {{openingTime}}.",
+    "eventStarted": "The event has already started and registration cannot be cancelled.",
+    "hasPaymentCancellation": "The payment will be cancelled and cannot be modified.",
```
Comment (harriris-vincit) [inline public/locales/en/signup.json:129]:
```suggestion
    "hasPaymentCancellation": "The payment is being cancelled and cannot be modified.",
```
Present tense could be used here like in the Finnish sentence.

File: public/locales/en/signup.json [hunk 2]
```diff
@@ -124,6 +124,10 @@
...
+    "hasPaymentRefund": "The payment will be refunded and cannot be modified.",
```
Comment (harriris-vincit) [inline public/locales/en/signup.json:130]:
```suggestion
    "hasPaymentRefund": "The payment is being refunded and cannot be modified.",
```

File: public/locales/sv/signup.json [hunk 3]
```diff
@@ -123,6 +123,10 @@
     "capacityInWaitingList_other": "Registrering för detta evenemang är fortfarande möjligt, men det är bara {{count}} platser kvar i kön.",
     "capacityInWaitingListNoLimit": "Registrering till detta evenemang är fortfarande möjlig, men det finns endast platser kvar i kön.",
     "closed": "Registreringen för detta evenemang är för närvarande stängd. Vänligen försök igen senare.",
-    "closedWithEnrolmentTime": "Anmälan till detta evenemang öppnar den {{openingDate}} kl {{openingTime}}."
+    "closedWithEnrolmentTime": "Anmälan till detta evenemang öppnar den {{openingDate}} kl {{openingTime}}.",
+    "eventStarted": "Evenemanget har redan börjat och registreringen kan inte avbrytas.",
+    "hasPaymentCancellation": "Registreringsavgiften kommer att annulleras och kan inte ändras.",
```
Comment (harriris-vincit) [inline public/locales/sv/signup.json:128]:
```suggestion
    "hasPaymentCancellation": "Anmälningsavgiften annulleras och kan inte ändras.",
```
Present tense form according to ChatGPT.

File: public/locales/sv/signup.json [hunk 4]
```diff
@@ -123,6 +123,10 @@
...
+    "hasPaymentRefund": "Anmälningsavgiften kommer att återbetalas och kan inte ändras.",
```
Comment (harriris-vincit) [inline public/locales/sv/signup.json:129]:
```suggestion
    "hasPaymentRefund": "Anmälningsavgiften återbetalas och kan inte ändras.",
```

**Review by harriris-vincit — APPROVED** [review]
Some suggestions about translations, but overall looks good!

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr168-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr168-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**